integration/tests/basic/test_rpc_calls.py

```diff
@@ -393,6 +393,44 @@ def test_get_rpc_logs_eq_val(self, event_caller_contract):
                                                 "blockNumber", "transactionIndex", "address",
                                                 "logIndex", "data", "transactionLogIndex"])
 
+    def test_eth_get_logs_list_of_addresses(self, event_caller_contract):
+        event_caller2, _ = (  # we need second contract to check list of addresses
+            self.web3_client.deploy_and_get_contract(
+                "EventCaller", "0.8.12", self.sender_account
+            ))
+
+        number = random.randint(1, 100)
+        text = "".join([random.choice(string.ascii_uppercase) for _ in range(5)])
+        bytes_array = text.encode().ljust(32, b'\0')
+        bol = True
+        tx = self.make_tx_object()
+        instruction_tx = event_caller_contract.functions.allTypes(  # transaction for first contract
+            self.sender_account.address, number, text, bytes_array, bol
+        ).build_transaction(tx)
+        self.web3_client.send_transaction(self.sender_account, instruction_tx)
+
+        tx2 = self.make_tx_object()
+        instruction_tx2 = event_caller2.functions.allTypes(  # transaction for second contract
+            self.sender_account.address, number, text, bytes_array, bol
+        ).build_transaction(tx2)
+        self.web3_client.send_transaction(self.sender_account, instruction_tx2)
+
+        params = {"address": [event_caller_contract.address, event_caller2.address]}  # list of addresses
+        topic = cryptohex("AllTypes(address,uint256,string,bytes32,bool)")
+        params["topics"] = [topic]
```

Comment:
You have executed 2 different contracts and waiting logs from 2 contracts, if filter doesn't work it will provide this events too. Send 2 transactions but filter for 1 contract address or add third contract

Reply:
we have tests with 1 contract address in filter
maybe there are just no checks about counts of logs

Reply:
@kristinaNikolaevaa ok, good

Reply:
👍